### DISCUSSION TITLE
Easy Spinbox for CustomTkinter

### DISCUSSION THREAD

**Discussion by CodeSame** [TomSchimansky/CustomTkinter#1737] [General]
Spinbox class for customtkinter. Easy to integrate into any project as a class. Supports minimum and maximum values, change values by scrolling the mouse wheel. Based on FloatSpinbox https://github.com/TomSchimansky/CustomTkinter. Hopefully someone will find this useful before the official realization in customtkinter
https://github.com/CodeSame/ctkSpinbox

**Comment by Akascape** [comment]
@CodeSame Nice.
You can also add a entry validate function so that it only write numbers.
**Something like this:**
```python
import customtkinter

def only_numbers(char):
    # Validate true for only numbers
    if (char.isdigit() or char==""):
        return True
    else:
        return False

root = customtkinter.CTk()

validation = root.register(only_numbers)

entry = customtkinter.CTkEntry(master=root, validate='key', validatecommand=(validation, '%P'))
entry.pack(padx=10, pady=10)

root.mainloop()
```

**Comment by ghost** [comment]
Move this to "Showcase"?
